### PR TITLE
Add variables for toc title border as well as toc title padding bottom

### PR DESCRIPTION
**Pull request by dannylonglegs**

### CODE REVIEW

File: assets/styles/variables/_section-titles.scss [hunk 1]
```diff
@@ -689,6 +689,43 @@ $toc-title-text-transform: $section-title-text-transform !default;
...
+$toc-title-border-bottom-width: (
+  epub: 0,
+  prince: 0,
+  web: 0
+) !default;
+
+/// Defines the bottom border style for the Table of Contents header.
+/// @type String | Map
+/// @since 1.0.0
+$toc-title-border-bottom-width: (
```
Comment (greatislander) [inline assets/styles/variables/_section-titles.scss]: This variable is wrong, which is why the test is failing.

**Comment by greatislander** [comment]
Depends on #139.

**Comment by dannylonglegs** [comment]
Can we close this branch? I believe I fixed the problem on the `front-matter-title-border` branch

**Comment by greatislander** [comment]
Superseded by #139.

**Pull request closed without merging**